tests/sandbox/create_message_batches/test_success.py

```diff
@@ -38,7 +46,15 @@ def test_201_message_batch_valid_accept_headers(nhsd_apim_proxy_url, accept_head
 @pytest.mark.parametrize('content_type', VALID_CONTENT_TYPE_HEADERS)
 def test_201_message_batch_valid_content_type_headers(nhsd_apim_proxy_url, content_type):
     """
-    .. py:function:: Test valid content type headers
+    .. py:function:: Scenario: A user submitting a request with a \
+        valid content type header receives a 201 response
+
+        | **Given** the user provides a valid content type header
```

Comment:
Given the user provides a valid content type header when creating a batch of messages